File: Data Structures & Algorithms/sort-colors/submission-8.java
class Solution {
    public void sortColors(int[] nums) {
        int l = 0;
        int i = 0;
        int r = nums.length - 1;
        while (i<=r){
            if (nums[i]==0){
                swap(nums,i,l);
                l++;

            }
            else if (nums[i]==2){
                swap(nums,i,r);
                r--;
                i--;
            }
            i++;
        }
        
    }
    public void swap(int[] nums,int i, int j){
        int temp = nums[i];
        nums[i] = nums[j];
        nums[j] = temp;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-8.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = []
        for i in range(len(nums)):
            revisedTarget = target - nums[i]
            for j in range(i+1,len(nums)):
                if nums[j] == revisedTarget:
                    result.append(i)
                    result.append(j)
        return result